src/main.js

```diff
@@ -209,22 +221,29 @@ async function getNonFollowersList(shouldIncludeVerifiedAccounts = true) {
             }
         });
 
-        elMainBtn.innerHTML = `${Math.round((currentFollowedUsersCount / totalFollowedUsersCount) * 100)}%`;
+        let percentage = `${Math.round((currentFollowedUsersCount / totalFollowedUsersCount) * 100)}%`;
+        loadingPercentage.innerHTML = percentage;
+        loadingPercentage.style.width = percentage;
         elResultsContainer.innerHTML = '';
         elNonFollowerCount.innerHTML = list.length.toString();
+        renderResults(list);
 
         await sleep(Math.floor(Math.random() * (1000 - 600)) + 1000);
         scrollCycle++;
         if (scrollCycle > 6) {
             scrollCycle = 0;
-            elResultsContainer.innerHTML =
-                '<span style="margin:1rem;">Sleeping 10 secs to prevent getting temp blocked...</span>';
+            sleepingContainer.style.display = 'inline-grid';
```

Comment:
Just as a reminder: if we go with the bottom version then this can just be `block` instead.